### PR TITLE
Evens out the odds mostly for Dynamic modes

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: hippiestation/code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm [hunk 1]
```diff
@@ -104,3 +104,30 @@
...
+	weight = 4
+	
+/datum/dynamic_ruleset/roundstart/monkey
+	weight = 0.2
+	requirements = list(101,101,70,40,30,20,10,10,10,10)
+	high_population_requirement = 10
+	flags = HIGHLANDER_RULESET
+	
+/datum/dynamic_ruleset/roundstart/nuclear
+	required_candidates = 2
```
Comment (Manly-Dwarf) [inline hippiestation/code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm]: required_candidates = 1

**Review by Manly-Dwarf — CHANGES_REQUESTED** [review]

File: hippiestation/code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm [hunk 2]
```diff
@@ -104,3 +104,30 @@
...
+/datum/dynamic_ruleset/roundstart/nuclear
+	required_candidates = 1
+
+/datum/dynamic_ruleset/roundstart/nuclear/clown_ops
+	requirements = list(90,90,90,80,60,40,30,20,10,10)
+	high_population_requirement = 10
+	weight = 0.3
+	
+/datum/dynamic_ruleset/roundstart/clockcult
+	cost = 35
```
Comment (Manly-Dwarf) [inline hippiestation/code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm]: I honestly would not mind Clockcult being the most expensive gamemode, due to its generally controversial nature and tendency to steamroll or get blown the fuck out in seconds any way.

Comment (Manly-Dwarf) [reply hippiestation/code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm]: They'll end the round no matter what anyhow.

**Comment by Manly-Dwarf** [comment]
so far the variety is good but the numbers are not, feels like it spawns just one too many antags every time. we need to find out the cause of that.
any way, I'm gonna look into merging this and the other PR, and getting it into the config

**Comment by Manly-Dwarf** [comment]
this weight though, I think going by double digits might fuck it up in some regard. hopefully not

**Comment by ghost** [comment]
i did a local test:
```
/mob/Login()
	..()
	var/list/A = list("a" = 40, "aa" = 35, "b" = 30, "bb" = 35, "c" = 20, "cc" = 25, "d" = 10, "e" = 2)
	for(var/i = 1 to 1000)
		var/B = pickweight(A)
		src << B
```

this was the result: https://pastebin.com/E3NE8rJ3

**Comment by Manly-Dwarf** [comment]
okay if I merge and this fails to compile that's your fault
we have butchered our coding standards for this gamemode and a couple other things, but that's because testmerging is broken

**Pull request merged**